src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -3301,7 +3301,12 @@ private function registerRateLimiterConfiguration(array $config, ContainerBuilde
 
             if (interface_exists(RateLimiterFactoryInterface::class)) {
                 $container->registerAliasForArgument($limiterId, RateLimiterFactoryInterface::class, $name.'.limiter');
-                $factoryAlias->setDeprecated('symfony/dependency-injection', '7.3', 'The "%alias_id%" autowiring alias is deprecated and will be removed in 8.0, use "RateLimiterFactoryInterface" instead.');
+                $factoryAlias->setDeprecated('symfony/dependency-injection', '7.3', \sprintf('The "%%alias_id%%" autowiring alias is deprecated and will be removed in 8.0, use "%s $%s" instead.', RateLimiterFactoryInterface::class, (new Target($name.'.limiter'))->getParsedName()));
+                $internalAliasId = \sprintf('.%s $%s.limiter', RateLimiterFactory::class, $name);
+
+                if ($container->hasAlias($internalAliasId)) {
```

Comment:
`registerAliasForArgument()` can register a second alias (that's what I call "internal alias" here): https://github.com/symfony/symfony/blob/ae30d7c04d249fd07758582d67fc5ea014363f9a/src/Symfony/Component/DependencyInjection/ContainerBuilder.php#L1475-L1477